src/lib.rs

```diff
@@ -202,6 +202,31 @@ impl<'a> MailHeader<'a> {
 
         result
     }
+
+    /// Get the raw, unparsed value of the header value.
```

Comment:
You have the descriptions and (partly) the examples backwards. This one should be "... of the header key" and the example should use `get_key_raw()` instead of `get_value_raw()`.

Reply:
Oh whoops, thank you!